climada/engine/unsequa/unc_output.py

```diff
@@ -557,6 +556,14 @@ def plot_uncertainty(self, metric_list=None, orig_list=None, figsize=None,
                 if ax is not None:
                     ax.remove()
                 continue
+            # Check if the column data is empty or contains only NaNs
+            data, m_unit = u_vtm(unc_df_plt[col])
+            data = pd.Series(data)
+            if data.empty or data.isna().all() or data.dropna().shape[0] < 2:
+                print(f"Skipping plot for '{col}': insufficient data.")
+                if ax is not None:
+                    ax.remove()
+                continue
```

Comment:
Is this part of this PR? This seems not related to the delta climate uncetainty.

Reply:
It is. I slightly modified the plotting functionalities of the uncertainty output to skip columns of the uncertainty output dataframe which are zeros or nan.

I added this change to the description of the PR.

Reply:
Good idea, but it looks a bit weird that the subplots are missing completely. Instead of removing the axes, you could make an empty plot and write a short text like "No data to plot"?